speculation-rules.bs

```diff
@@ -217,6 +233,85 @@ Inside the [=prepare the script element=] algorithm we make the following change
     </dd>
   </dl>
 
+<h3 id="speculation-rules-header">The [:Speculation-Rules:] header</h3>
+Note: This section contains modifications to [[HTML]].
+
+<div algorithm="process the Speculation-Rules header">
```

Comment:
You can just write `<div algorithm>` btw, it automatically looks for the `<dfn>` to form a default name.

Reply:
Done